tower-grpc/src/lib.rs

```diff
@@ -1,4 +1,10 @@
-#![deny(warnings, missing_debug_implementations, rust_2018_idioms)]
+#![doc(html_root_url = "https://docs.rs/tower-grpc/0.1.0")]
+#![deny(missing_debug_implementations, rust_2018_idioms)]
+// TODO: enable when there actually are docs
+// #![deny(missing_docs)]
```

Comment:
TODO?

Reply:
Many types don’t have docs. Also, I expect the vast majority will go away in 0.2 when we switch to async / await. I rather not spend time on it now. 

This release is mostly about just putting what we have on crates.io

Reply:
fine with me!